clusterdef/nodegroup.go

```diff
@@ -21,6 +21,10 @@ type DockerNodeGroup struct {
 
 type CloudNodeGroup struct {
 	InstanceType string `yaml:"instance-type,omitempty"`
+	Cpu          int    `yaml:"cpu,omitempty"`
+	MemoryInGB   int    `yaml:"memory-in-gb,omitempty"`
+	ServerImage  string `yaml:"server-image,omitempty"`
+	ReleaseId    string `yaml:"release-id,omitempty"`
```

Comment:
What is release-id?

Reply:
releaseId is id for the image release
7.2 image have release id - "1.0.29"
7.6 image have release id - "1.0.28"

Reply:
Release id needs to be passed while updating server version in modify command

Reply:
Is the release-id not part of the server-image?  It seems like we should be able to parse the release-id out of the server image trivially rather than requiring it to be specified on its own.  Alternatively, we could use only the release-id here (with no server-image) and use the "build-number" capabilities of the node version to create a server-image path.

Reply:
We can get release id from server-image, as each cloud images have this